sections/main-404.liquid

```diff
@@ -7,6 +7,7 @@
   </header>
 </div>
 
+<script>window.location.replace("/collections/performance-sandbags");</script>
```

Comment:
place this scrip on top of the 404 templates so that it redirects immediately, instead of loading the 404 page first and then redirect to the shop page.